### ISSUE TITLE
Parameter Insert generates illegal parameter names

### ISSUE THREAD

**Issue by gab1one** [knime-ip/knip-scripting#34]
- illegal characters not stripped:
  - eg: [ ( /  $% . 
- duplication avoidance only after compilation

**Comment by Squareys** [comment]
Neat! Has the duplication avoidance without recompilation been fixed also?

**Comment by gab1one** [comment]
Not yet, that is a separate issue.
